nemo/collections/nlp/modules/common/token_classifier.py

```diff
@@ -73,6 +74,7 @@ def __init__(
         self.mlp = MultiLayerPerceptron(
             hidden_size, num_classes, num_layers=num_layers, activation=activation, log_softmax=log_softmax
         )
+        self.__hidden_size = hidden_size
```

Comment:
could you change to _hidden_size instead of __hidden_size? also everywhere else in PR?

Reply:
changed